### DISCUSSION TITLE
Double click for editing a card

### DISCUSSION THREAD

**Discussion by laughedelic** [mgmeyers/obsidian-kanban#64] [Ideas & Feature Requests]
Hi! First, thanks for this plugin, it's awesome!

I find it a bit inconvenient to use the context menu of the card to reach to the edit button. Would it be possible to trigger card editing by a double click on it?

**Comment by mgmeyers** [comment]
Thanks @laughedelic! As of yesterday you can now ctrl/cmd + click on the card text to edit cards. Does this work for your use case?

**Reply by laughedelic** [reply]
Thank you! This is definitely better than clicking on the context menu 👍 

I'm sorry for creating this separate duplicate discussion, I just noticed that there was already one: https://github.com/mgmeyers/obsidian-kanban/discussions/34. I searched in issues before creating, but haven't searched properly in Discussions 😓 

Yet this shows that different users independently came to the same feature request. And now I see another one: https://github.com/mgmeyers/obsidian-kanban/discussions/5#discussioncomment-711655.

I think that double click is a more intuitive gesture for interacting with the cards than a hot-key plus click which requires two hands and isn't that common (at least in my experience) in other interfaces. Would you consider enabling double-click as well or making it configurable?

**Reply by mgmeyers** [reply]
No problem on the duplicate discussion! I'm glad you offered your take on it.

The one downside of double click, and the reason I initially chose not to implement it, is that there are some complications when cards contain links. Eg: you can't double click on a link without it opening. That being said, I'll play around with double clicking and see if I can find a way to implement it that doesn't feel too awkward in these cases.

**Reply by laughedelic** [reply]
I see, thanks for the explanation. It's great to know that you've considered it and that your decisions are driven by usability!

**Reply by danieltomasz** [reply]
@mgmeyers Thanks for bringing  to life the  idea of this plugin and its  development;

 I think this  thread solve my riddle from here https://github.com/mgmeyers/obsidian-kanban/issues/78 

In Obsidian in preview mode control +click is generally used to open a note in new pane, just clicking on wiki link should open the link in pane (or new pane if the note is pinned); this behaviour is different when using the latest kanban plugin, clicking will always open link in new pane

I treat kanban note as pinned master  note  and I try to use card as task-notes link to standalone card; without possibility to open-in-place wiki -inks soon my obsidian workspace is too crammed with many copies of the same note in different panes;

My temporary solution I would propose is to allow a toggle for users to turn off and on this Ctrl +click behaviour, so users who want standard Obsidian behaviour for opening the links will have a workaround until a better implementation will appear

Another solution, I don't know if this is taken, but maybe Alt/Option + click for editing card?

EDIT: I tried  reverting to 0.2.3, but this doesn't solve my problem, both click and Ctrl+click  on link elicit the same behaviour (open note in new pane)

**Reply by mgmeyers** [reply]
@danieltomasz Yeah, there are currently some issues related to links and which pane they open in. I'll be fixing this shortly.

**Reply by fuhadabdulla** [reply]
Thank you mgmeyes double click to edit is now available it is very convinient thank you

**Comment by mgmeyers** [comment]
@laughedelic I've decided to go with double click after all, rather than cmd/ctrl + click. This should work in the latest update.

**Reply by laughedelic** [reply]
Great news! Thanks!

**Reply by fuhadabdulla** [reply]
Thank you so much
